internal/blueprint/customizations.go

```diff
@@ -66,9 +66,9 @@ type ServicesCustomization struct {
 }
```

Comment:
This commit should probably be squashed into the previous one :)

Reply:
@teg okay sure thing, I will extend those tests. Could you please point me in the correct direction for the CLI tests please? =)